MV.Wildebeest.Core/source/core/java/co/mv/wb/impl/WildebeestApiImpl.java

```diff
@@ -740,43 +738,80 @@ private static void throwIfFailed(
 		}
 	}
 
-	private static void findPaths(
+	public static List<List<Migration>> findPaths(
```

Comment:
Made this method public but didn't add parameter guards - lets add if(fromState == null) throw new ArgumentNullException etc

Reply:
Thanks.  I forgot to add those.  I have added them on my latest pushed.